pyproject.toml

```diff
@@ -178,6 +180,7 @@ include = [
     "etc/config/metadata/runtime-images/*.json",
     "etc/config/metadata/component-registries/*.json",
     "etc/config/settings/*.json",
+    "build/labextensions",
```

Comment:
Did moving this actually make a difference?  I only raise this because, otherwise, these are in alphabetical order implying their order really doesn't matter.  But someone looking at this may wonder if there's some kind of magic or _why_ `build/labextensions` needs to be last.  

If the order does matter, could you please add a comment, otherwise I think we should restore back to purely alphabetical until we find a need to reorder things?

Reply:
Reordered the data file listing to original and added instructions to the top on how to build using the release script.